### PR TITLE
Chroma vector store

### PR DESCRIPTION
**Pull request by Chitti-Ankith**
Added support for ChromaDB in Eva. Will focus on benchmarking the existing vector stores after this.

### CODE REVIEW

**Comment by xzdandy** [comment]
Hi @Chitti-Ankith , thanks for adding Chroma vector store support. Is this PR ready for review?

**Comment by Chitti-Ankith** [comment]
Hi @xzdandy yes it is, can you please have a look?

File: test/integration_tests/long/test_similarity.py [hunk 1]
```diff
@@ -411,6 +411,24 @@ def test_end_to_end_index_scan_should_work_correctly_on_image_dataset_qdrant(sel
         # Cleanup
         self.evadb.catalog().drop_index_catalog_entry("testQdrantIndexImageDataset")
 
+    @chromadb_skip_marker
+    def test_end_to_end_index_scan_should_work_correctly_on_image_dataset_chromadb(
```
Comment (xzdandy) [inline test/integration_tests/long/test_similarity.py:415]: Please update the circle ci or dev configuration, so the test will not be skipped.

**Review by xzdandy — COMMENTED** [review]
Overall looks good to me. Request @jiashenC 's review who has more expertise. We need to add a documentation page to summarize all vector database we supported and their available configuration parameters.

File: test/integration_tests/long/test_similarity.py (hunk 1 above)
Comment (Chitti-Ankith) [reply test/integration_tests/long/test_similarity.py:415]: My bad, fixed the circleci config

**Comment by Chitti-Ankith** [comment]
> Overall looks good to me. Request @jiashenC 's review who has more expertise. We need to add a documentation page to summarize all vector database we supported and their available configuration parameters.

I'll add the documentation once the benchmark is done

**Review by jiashenC — CHANGES_REQUESTED** [review]
It looks good overall. You might miss a test case in `test_similarity`

**Comment by Chitti-Ankith** [comment]
> It looks good overall. You might miss a test case in `test_similarity`

I added the chromadb requirement in circleci, can you tell me which test case will be missed?

**Comment by jiashenC** [comment]
> > It looks good overall. You might miss a test case in `test_similarity`
> 
> I added the chromadb requirement in circleci, can you tell me which test case will be missed?

NVM, I misread your test cases. I think it is good to merge.

**Review by jiashenC — APPROVED** [review]

**Pull request merged**